fairlearn/datasets/_fetch_data.py

```diff
@@ -0,0 +1,91 @@
+import warnings
```

Comment:
Perhaps we should have individual files per dataset? A single _fetch_data file for all datasets may get super big soon and then hard to read. What do you think?

Reply:
I agree seperating is best. Made the improvement locally.